code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -226,6 +226,27 @@ var/list/ai_verbs_default = list(
 
 	job = "AI"
 
+/mob/living/silicon/ai/Stat()
+	..()
+	if(statpanel("Status"))
+		if(!stat)
+			show_borg_info()
+			return
+		stat(null, text("Systems nonfunctional"))

```

Comment:
This should be the other way around tbh, `if(stat) stat(null) return`

Reply:
Everything being named stat is so confusing...hopefully I fixed it the right way!